enterprise/server/usage/usage.go

```diff
@@ -79,16 +78,23 @@ var (
 )
 
 type TrackerOpts struct {
+	// TEST ONLY options
+
+	// Clock overrides the clock used by the usage tracker.
 	Clock timeutil.Clock
+
+	// FlushLock overrides the distributed lock to serialize access to the DB
+	// across flush jobs.
+	FlushLock interfaces.DistributedLock
```

Comment:
nit: can you remove this from the opts and make it mandatory (like, accept it as the second param in the NewTracker func?)

Reply:
Done